docs/spark-standalone.md

```diff
@@ -751,8 +751,7 @@ Learn more about getting started with ZooKeeper [here](https://zookeeper.apache.
 
 **Configuration**
 
-In order to enable this recovery mode, you can set SPARK_DAEMON_JAVA_OPTS in spark-env by configuring `spark.deploy.recoveryMode` and related spark.deploy.zookeeper.* configurations.
-For more information about these configurations please refer to the [configuration doc](configuration.html#deploy)
```

Comment:
The `#deploy` section doesn't exist anymore. Neither does documentation for the `spark.deploy.*` configs mentioned here.